dolphin/db/sqlalchemy/api.py

```diff
@@ -114,10 +114,28 @@ def storage_create(storage):
     return storage_ref
 
 
+def registered_device_list(storage):
```

Comment:
use Registry_contet_craete API for registration create. you can add fileds in registry_contect if required

Reply:
ok done.